### PR TITLE
fix(ui): use Label component

### PR DESCRIPTION
**Pull request by joseivanlopez**
Drop `Tag` component in favor of patternfly `Label`.

Screenshot from 2024-06-10 15-46-03

### CODE REVIEW

File: web/src/components/storage/ProposalResultTable.jsx [hunk 1]
```diff
@@ -89,21 +90,21 @@ const DeviceCustomSize = ({ item, devicesManager }) => {
   const sizeBefore = isResized ? devicesManager.systemDevice(device.sid).size : item.size;
 
   return (
-    <div className="split">
+    <Flex direction={{ default: "row" }} gap={{ default: "gapXs" }}>
+      <DeviceSize item={item} />
       <If
         condition={isResized}
         then={
-          <Tag variant="orange">
```
Comment (dgdavid) [inline web/src/components/storage/ProposalResultTable.jsx:96]: Thus, <Tag> component is now only used by storage/device-utils.jsx. Something to check and drop

File: web/src/assets/styles/blocks.scss [hunk 2]
```diff
@@ -510,31 +510,6 @@ table.proposal-result {
     );
   }
   /** End of temporary hack */
-
-  @media (width > 768px) {
-    th.details-column {
-      padding-inline-start: calc(60px + var(--spacer-smaller) * 2);
-    }
-
-    td.details-column {
```
Comment (dgdavid) [inline web/src/assets/styles/blocks.scss:519]: Do you mind dropping only the reference to these CSS classes in ProposalResultSection.jsx when defining the const `columns`? Thanks in advance.

**Review by dgdavid — COMMENTED** [review]
LGTM, just minor comments

File: web/src/components/storage/ProposalResultTable.jsx (hunk 1 above)
Comment (joseivanlopez) [reply web/src/components/storage/ProposalResultTable.jsx:96]: Dropped.

**Review by dgdavid — APPROVED** [review]

**Pull request merged**